functions.php

```diff
@@ -33,14 +46,17 @@ function connect_db_return_collection($collectionName)
 function eachCharacter(array $character): string
 {
     $characterHtmlElementsString = '';
-    $characterImageElement = '<img src="' . $character['image'] . '" alt="Picture of ' . $character['character_name'] . '">';
-    $characterNameElement = $character['character_name'];
-    $characterTvShowElement = $character['name'];
-    $characterIqElement = $character['IQ'];
-    $characterHtmlElement = '<div class="container">' . $characterImageElement . "<h2>Name: " . $characterNameElement . '</h2>' . "<h2>TV Show: " . $characterTvShowElement . '</h2>' . "<h2>IQ: " . $characterIqElement . '</h2>' . '</div>';
-    $characterHtmlElementsString .= $characterHtmlElement;
-    if (!isset($character['character_name']) || !isset($character['IQ']) || !isset($character['name']) || !isset($character['image'])) {
-        return '';
-    };
+    if (isset($character['character_name']) &&
```

Comment:
almost. PSR2 says:
```
if (
    condition &&
    condition
) {
```
first condition should be on its own line, not inline with the if